web-data/data/browsers.css-data.json

```diff
@@ -447,12 +447,6 @@
       ],
       "values": [],
       "relevance": 50,
-      "references": [
-        {
-          "name": "MDN Reference",
-          "url": "https://developer.mozilla.org/docs/Web/CSS/alt"
-        }
-      ],
```

Comment:
Don't we want these references? Was this removed by someone else and we're just syncing?